src/test/regress/sql/propagate_alter_role.sql

```diff
@@ -0,0 +1,86 @@
+CREATE SCHEMA alter_role;
+
+SET citus.enable_alter_role_propagation to ON;
+SELECT run_command_on_workers($$SET citus.enable_alter_role_propagation to ON;$$);
+
+CREATE ROLE alter_role_1 WITH LOGIN;
+SELECT run_command_on_workers($$CREATE ROLE alter_role_1 WITH LOGIN;$$);
+
+-- postgres errors out
+ALTER ROLE alter_role_1 WITH SUPERUSER NOSUPERUSER;
+
+-- make sure that we propagate all options accurately
+ALTER ROLE alter_role_1 WITH SUPERUSER CREATEDB CREATEROLE INHERIT LOGIN REPLICATION BYPASSRLS CONNECTION LIMIT 66 VALID UNTIL '2032-01-01 00:00:00+00'; 
+SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, rolvaliduntil) FROM pg_authid WHERE rolname = 'alter_role_1';
+SELECT run_command_on_workers($$SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, rolvaliduntil) FROM pg_authid WHERE rolname = 'alter_role_1'$$);
+
+-- make sure that we propagate all options accurately
+ALTER ROLE alter_role_1 WITH NOSUPERUSER NOCREATEDB NOCREATEROLE NOINHERIT NOLOGIN NOREPLICATION NOBYPASSRLS CONNECTION LIMIT 0 VALID UNTIL '2052-01-01 00:00:00+00'; 
+SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, rolvaliduntil) FROM pg_authid WHERE rolname = 'alter_role_1';
+SELECT run_command_on_workers($$SELECT row(rolname, rolsuper, rolinherit,  rolcreaterole, rolcreatedb, rolcanlogin, rolreplication, rolbypassrls, rolconnlimit, rolpassword, rolvaliduntil) FROM pg_authid WHERE rolname = 'alter_role_1'$$);
```

Comment:
Please update all the occurrences of `rolvaliduntil` in the SQL test files -- including isolation tests -- with `left(rolvaliduntil::text, 10)`  to ignore the hour,  only get the `YYY-MM-DD`.
Otherwise, the tests are dependent on the timezone

Reply:
Or, maybe  just use ` EXTRACT (year FROM rolvaliduntil)`, and set to different `years` in the tests. So,  seeing the correct year is  enough